.github/workflows/lint.yaml

```diff
@@ -0,0 +1,20 @@
+name: Lint (Hadolint)
```

Comment:
A comment before this file would be helpful, perhaps something like:
`# Linter for CI for patroni docker image`

Also not sure if its required but you might want the:
```
# Copyright 2021 Canonical Ltd.
# See LICENSE file for licensing details.
```

Reply:
Great addition, Mia. I believe that it's not required to add the copyright notice, but looks better to have it. I also added the LICENSE file as we are commenting about it in this header.